ooniprobe/Utility/DictionaryUtility.h

```diff
@@ -0,0 +1,15 @@
+//
+//  DictionaryUtility.h
+//  ooniprobe
+//
+//  Created by Lorenzo Primiterra on 06/08/17.
+//  Copyright © 2017 Simone Basso. All rights reserved.
+//
```

Comment:
Can you update the comment? (The presence of this comment, strictly speaking, make ooniprobe-ios non FLOSS)

Reply:
^^^ @lorenzoPrimi